src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArtworksRail.tsx

```diff
@@ -22,6 +22,19 @@ export type DiscoveryArtwork = {
   imageUrl: string
 }
 
+function getPriceRange(budgetIntent?: BudgetIntent) {
+  let priceMinUSD = budgetIntent?.budget?.min
+  let priceMaxUSD = budgetIntent?.budget?.max
+  let numberOfArtworks = budgetIntent?.numberOfArtworks
+
+  if (numberOfArtworks) {
+    if (priceMinUSD) priceMinUSD = priceMinUSD / numberOfArtworks
+    if (priceMaxUSD) priceMaxUSD = priceMaxUSD / numberOfArtworks
+  }
```

Comment:
We make use of the inferred `numberofArtworks` as well (though not yet the `timeFrame`)

Reply:
I'm very curious to see how well it does determining a the `numberOfArtworks` after more experimenting.